runtime-testsuite/test/org/antlr/v4/runtime/TestCodePointCharStream.java

```diff
@@ -3,17 +3,13 @@
  * Use of this file is governed by the BSD 3-clause license that
  * can be found in the LICENSE.txt file in the project root.
  */
-package org.antlr.v4.test.runtime.java;
+package org.antlr.v4.runtime;
```

Comment:
Good spot! I did this on purpose so I could keep the API `CodePointBuffer.getType()` package-private. (That's a common pattern to allow tests but not the public to access certain APIs for testing).